en/components/banner.md

```diff
@@ -238,91 +180,153 @@ export class MyBannerComponent {
+
+If the component is using the [`Emulated`](themes/component-themes.md#view-encapsulation) ViewEncapsulation, it is necessary to `penetrate` this encapsulation using `::ng-deep`. To prevent the custom theme to leak into other components, be sure to include the `:host` selector before `::ng-deep`:
 
 ```scss
-// in component.scss
 :host {
     ::ng-deep {
-        // Pass our banner theme to the `igx-banner` mixin
+        // Pass the custom banner theme to the `igx-banner` mixin
         @include igx-banner($custom-banner-theme);
-        // Pass our button theme to the `igx-button` mixin
-        @include igx-button($custom-button-theme);
     }
 }
 ```
-### Styling Demo
+
+#### Using color palettes
+
+Instead of hardcoding the color values, like we just did, we can achieve greater flexibility in terms of colors by using the [`igx-palette`]({environment:sassApiUrl}/index.html#function-igx-palette) and [`igx-color`]({environment:sassApiUrl}/index.html#function-igx-color) functions.
+
+`igx-palette` generates a color palette based on the primary and secondary colors that are passed:
+
+```scss
+$white-color: #dedede;
+$black-color: #151515;
+
+$light-banner-palette: igx-palette($primary: $white-color, $secondary: $black-color);
+```
+
+And then with [`igx-color`]({environment:sassApiUrl}/index.html#function-igx-color) we can easily retrieve color from the palette. 
+
+```scss
+$custom-banner-theme: igx-banner-theme(
+    $banner-message-color: igx-color($light-banner-palette, "secondary", 400),
+    $banner-background: igx-color($light-banner-palette, "primary", 400),
+    $banner-illustration-color: igx-color($light-banner-palette, "secondary", 100)
+);
+```
+
+>[!NOTE]
+>The `igx-color` and `igx-palette` are powerful functions for generating and retrieving colors. Please refer to the [`Palettes`](themes/palette.md) topic for detailed guidance on how to use them.
+
+#### Using schemas
+
+You can build a robust and flexible structure that benefits from [`schemas`](themes/schemas.md).
+Extend one of the two predefined schemas, that are provided for every component, in this case - [`light-banner`]({environment:sassApiUrl}/index.html#variable-_light-banner) schema: 
+
+```scss
+//  Extending the banner schema
+$light-toast-schema: extend($_light-toast,
+    (
+        banner-message-color: (
+            igx-color: ("secondary", 400)
+        ),
+        banner-background: (
+            igx-color: ("primary", 400)
+        ),
+        banner-illustration-color: (
+            igx-color: ("secondary", 100)
+        )
+    )
+);
+```
+
+In order to apply our custom schemas we have to **extend** one of the globals ([`light`]({environment:sassApiUrl}/index.html#variable-light-schema) or [`dark`]({environment:sassApiUrl}/index.html#variable-dark-schema)), which is basically pointing out the components with a custom schema, and after that add it to the respective component themes:
```

Comment:
This is no longer the case. You can simply pass the custom `$light-toast-schema` to the banner theme.